screens/MainLayout.tsx

```diff
@@ -35,6 +38,7 @@ export const MainLayout: React.FC<RootRouteProps> = () => {
           component={route.component}
           options={{
             ...route.options,
+            title: t(route.name.toLowerCase()).toUpperCase(),
```

Comment:
I this intended? isn't toUpperCase would still transform text to uppercase without lower case right?

Reply:
The lowercase transform is for the translation key. Then uppercase according to the design spec for the header text.